www/src/data/sidebars/doc-links.yaml

```diff
@@ -92,6 +92,8 @@
         - title: Sourcing from (headless) CMSs
           link: /docs/headless-cms/
           items:
+            - title: Sourcing from ButterCMS
+              link: /docs/sourcing-from-buttercms/
```

Comment:
We're trying to keep the "above-the-fold" links to 5, as linked by the most popular source plugins in https://www.gatsbyjs.org/plugins/?=gatsby-source- (with a bump to WP and Drupal since they're globally popular CMSs). So unfortunately, we can't accept this link. 

Instead, you should add it to the "Other CMS systems" section -- you can change to `ButterCMS (docs, guide, starter)`

Reply:
@calcsam ok thanks